layouts/partials/head/ogp.html

```
@@ -1,14 +1,14 @@
 {{ $imagePath := "" }}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)